packages/components/src/entities/SampleTypeDesignPage.tsx

```diff
@@ -101,6 +108,11 @@ export const SampleTypeDesignPage: FC<Props> = memo(props => {
 
     const schemaQuery = useMemo(() => new SchemaQuery(SCHEMAS.SAMPLE_SETS.SCHEMA, queryName), [queryName]);
 
+    const freezerManagementEnabled = isFreezerManagementEnabled(moduleContext);
+    const systemFields = freezerManagementEnabled
```

Comment:
any reason to wrap this in `useMemo`?

Reply:
Assuming this is actually about why it's not in a `useMemo`—while it won't add too much of a boost it doesn't hurt to wrap it, so I've updated 👍